tests/e2e/specs/wcpay/merchant/merchant-admin-multi-currency.spec.js

```diff
@@ -20,4 +21,31 @@ describe( 'Admin Multi-Currency', () => {
 		} );
 		await takeScreenshot( 'merchant-admin-multi-currency' );
 	} );
+
+	it( 'should be possible to add the currency switcher to the sidebar', async () => {
+		await merchantWCP.addMulticurrencyWidget();
+	} );
+
+	it( 'should be possible to add the currency switcher to a post/page', async () => {
+		await page.goto( `${ config.get( 'url' ) }wp-admin/post-new.php` );
```

Comment:
`WP_ADMIN_NEW_POST_TYPE` didn't work because it ends with `post-new.php?post_type=` and errors. I changed it to `WP_ADMIN_DASHBOARD + post-new.php` in https://github.com/Automattic/woocommerce-payments/pull/7822/commits/2c2259a08fca72e0cf15b911d1bb8d6dba60dba6.